buildtap.py

```diff
@@ -407,13 +409,21 @@ def inf2cat(self, x64):
         self.system("%s /driver:%s /os:%s" % (self.inf2cat_cmd, self.drvdir(self.src, x64), oslist))
 
     def sign(self, file):
-            self.system("%s sign /v /ac %s /s my /n %s /t %s %s" % (
-                    self.signtool_cmd,
-                    self.crosscert,
-                    self.sign_cn,
-                    self.timestamp_server,
-                    file,
-                ))
+        certspec = ""
+        if self.sign_cert:
+            certspec += "/f %s " % self.sign_cert
+            if self.cert_pw:
+                certspec += "/p %s " % self.cert_pw
+        else:
+            certspec += "/s my /n %s " % self.sign_cert
```

Comment:
Shouldn't this line say 

    certspec += "/s my /n %s " % self.sign_cn

instead? If self.sign_cert is not defined, then %s will be empty.